lib/jxl/encode.cc

```diff
@@ -191,6 +207,21 @@ JxlEncoderStatus JxlEncoderSetBasicInfo(JxlEncoder* enc,
       return JXL_ENC_ERROR;
       break;
   }
+  if (info->have_animation) {
+    enc->metadata.m.have_animation = true;
+    enc->metadata.m.animation.have_timecodes = false;
```

Comment:
the num_loops field should also be copied:

enc->metadata.m.animation.num_loops = info->animation.num_loops